api/envoy/type/matcher/v3/regex.proto

```diff
@@ -45,6 +45,8 @@ message RegexMatcher {
     //
     // This field is deprecated; regexp validation should be performed on the management server
     // instead of being done by each individual client.
+.. note::
```

Comment:
@ankatare im assuming this should have been indented inside the comment, no ?

Reply:
@phlax Ahh. Sorry. 
it should be like...  as below ?  
//.. note::
  //   The program size is still checked against the runtime value.

Reply:
i think the indent *of* the comment should be the same, the `.. note::` probs requires a single space indent, and the following line, not sure, but i think 4 indents

you might need another comment line underneath, also not sure about that

Reply:
humm, let me check and run first it locally.. Thanks

Reply:
Done